src/main/java/ReadWriteFile.java

```diff
@@ -0,0 +1,73 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class ReadWriteFile {
+
+    public static void readTaskList(ArrayList<Task> taskList) throws DukeException {
+        java.nio.file.Path path = java.nio.file.Paths.get("data").resolve("duke.txt");
+        boolean dirExists = java.nio.file.Files.exists(path);
+
+        if (dirExists) {
+            try {
+                File inFile = new File(path.toString());
+                Scanner sc = new Scanner(inFile);
+                Task t;
+
+                while (sc.hasNext()) {
+                    String[] task = sc.nextLine().split("\\|");
+                    switch (task[0].trim()) {
+                        case "T":
```

Comment:
Same issue pointed out by the first reviewer. Take note that there's no indentation for your case blocks.